x/compliance/handler_test.go

```diff
@@ -341,6 +341,80 @@ func TestHandler_ProvisionDifferentModels(t *testing.T) {
 	}
 }
 
+func TestHandler_UpdateComplianceInfoByCertificationCenter(t *testing.T) {
```

Comment:
Please add the following tests here:
- NewMsgUpdateComplianceInfo with no fields updated (all optional fields are not set). Make sure that all old values are kept.
- NewMsgUpdateComplianceInfo with all fields updated. Make sure that all updated fields have changed. 
- NewMsgUpdateComplianceInfo with CdCertificationID changed. Make sure that quering by CdCertificationID works as expected.